### ISSUE TITLE
Cannot log in with Facebook: "Missing client_id parameter."

### ISSUE THREAD

**Issue by anikolaienko** [lepture/authlib#5]
From authorize_access_token call I receive:
'{"error":{"message":"Missing client_id parameter.","type":"OAuthException","code":101,"fbtrace_id":"AkqIKeKkCiT"}}'

During debug I found in oauth.py fetch_access_token(...), when body object is formatted it has one of the query parameters "client=....", which in fact should be "client_id=..." for Facebook.
Unfortunately, there is no compliance_hook here for it.

I would suggest to make a complience_hook for it. In fact I want to contribute to authlib by myself.

**Comment by anikolaienko** [comment]
In fact, LinkedIn authorization doesn't work for same reason, but also it requires me to add explicitly client_secret to LinkedIn request.

**Comment by lepture** [comment]
@anikolaienko You can pass the value in `fetch_access_token`:

```python
>>> sess.fetch_access_token(URL, code=code, method='GET', client_id=sess.client_id)
```

But yes, it would be convenience to have a compliance hook.

**Comment by lepture** [comment]
@anikolaienko 

> when body object is formatted it has one of the query parameters "client=....", which in fact should be "client_id=..." for Facebook.

This is a bug. Thanks for reporting.

**Comment by lepture** [comment]
@anikolaienko I've released a bug fix version: 0.2.1.

**Comment by anikolaienko** [comment]
@lepture many thanks. client_id definitely fixed the problem.
I see fetch_access_token parameters, but since I am using Flask can we introduce the changes in my merge request: https://github.com/lepture/authlib/pull/6
